randovania/games/fusion/logic_database/Sector 5 ARC.txt

```diff
@@ -299,17 +299,21 @@ Extra - room_id: [9]
   * L3 Hatch to Security Save Room/Door to Security Shaft West
   * Extra - door_idx: (18,)
   > Door to Level 3 Security Room
-      # TODO: dmg run
-      Varia Suit
+      Any of the following:
+          # Damage Reqs
+          Varia Suit
+          Damage Runs (Expert) and Cold Damage ≥ 366
 
 > Door to Level 3 Security Room; Heals? False
   * Layers: default
   * L3 Hatch to Level 3 Security Room/Door to Security Shaft West
   * Extra - door_idx: (20,)
   > Door to Security Save Room
       All of the following:
-          # TODO:  damage reqs
-          Varia Suit
+          Any of the following:
+              # Damage Reqs
+              Varia Suit
+              Hi-Jump and Space Jump and Damage Runs (Expert) and Cold Damage ≥ 600
```

Comment:
how much damage do you take by only doing this with WJ?
also why is there HJ+SJ?